### PR TITLE
Remove Ethereum and Fabric references.

### PR DESCRIPTION
**Pull request by carrielaben-da**
The "Setting Up a Synchronization Domain" section of the Installation page needs to be checked/updated to account for the native Canton driver.

### CODE REVIEW

**Review by garyverhaegen-da — APPROVED** [review]
Approving to unblock, but please get a review from the Canton team too as I'm not familiar with the subject matter here.

**Comment by garyverhaegen-da** [comment]
You'll need to edit the `_toc.yml` file to match the changed structure to get this to build:

```patch
diff --git a/docs/3.1/_toc.yml b/docs/3.1/_toc.yml
index 0e6de80aaf..638cf12b83 100644
--- a/docs/3.1/_toc.yml
+++ b/docs/3.1/_toc.yml
@@ -1186,17 +1186,6 @@ subtrees:
         title: "Work With the Docker Containers"
       - file: canton/usermanual/static_conf
         title: "Set Configuration Options"
-      - file: canton/usermanual/domains/domains
-        title: "Set Up A Daml Domain With Enterprise Drivers"
-        subtrees:
-        - hidden: False
-          entries:
-          - file: canton/usermanual/domains/oracle
-            title: "Oracle Domain"
-          - file: canton/usermanual/domains/fabric
-            title: "Fabric Domain"
-          - file: canton/usermanual/domains/ethereum
-            title: "Ethereum Domain"
   - file: deploy-daml/infrastructure-architecture/high-availability/main-menu
     subtrees:
     - hidden: False
@@ -1533,13 +1522,6 @@ subtrees:
           title: "Contract Keys in Canton"
         - file: canton/architecture/domains/domains
           title: "Domain Architecture and Integrations"
-          subtrees:
-          - hidden: False
-            entries:
-            - file: canton/architecture/domains/ethereum
-              title: "Canton Domain on Ethereum"
-            - file: canton/architecture/domains/fabric
-              title: "Canton Domain on Fabric"
         - file: canton/architecture/identity
           title: "Identity Management"
         - file: canton/architecture/requirements/requirements
```

**Comment by garyverhaegen-da** [comment]
I can push that change to this branch if it makes it easier for you.

**Review by rgugliel-da — APPROVED** [review]

**Pull request merged**
